engine/runtime-compiler/src/main/scala/org/enso/compiler/pass/PassManager.scala

```diff
@@ -18,7 +22,7 @@ import org.enso.compiler.core.CompilerError
   */
 //noinspection DuplicatedCode
 class PassManager(
-  passes: List[PassGroup],
+  protected val passes: List[PassGroup],
```

Comment:
`protected`!? Is there a subclass of `PassManager`?

Reply:
Made private in 3e1ade9fff64052528ddc436213675b14c458bc2 . Note that it has to stay as field - we need it to be a field so that it is accessible in tests via https://github.com/enso-org/enso/blob/fb3e29d87b1328ca8bb78ef73c76421cc1e6db8f/engine/runtime-integration-tests/src/test/scala/org/enso/compiler/test/CompilerTest.scala#L34-L46

Reply:
EDIT: It has to be protected as it is accessed in tests in [PassManagerTestUtils](https://github.com/enso-org/enso/blob/546f17cde22689d78255def45038d13a96d25ebe/engine/runtime-instrument-common/src/test/java/org/enso/compiler/pass/PassManagerTestUtils.java#L9).